### ISSUE TITLE
Crash upon leaving Wyrm's Lookout and loading into Wyrm's Crossing (End of Act 2/Start of Act 3)

### ISSUE THREAD

**Issue by FrenchiestOfFries** [Norbyte/bg3se#403]
### I have attempted solutions to common problems

Yes

### I have checked that my game version is supported

Yes

### OS

Windows 10

### Platform

Steam

### Description

Attempting to leave camp after the "intro" to Act 3 (Astral Plane Night), I am greeted with the worst bugs I have encountered in the game in my ~550 hours of gameplay. After the night is finished with no errors to note, I proceed to talk to all of the companions at camp, no problems there, do a little respec'ing/swapping builds around, egregiously quick saving after every step as I hate having to re-do progressions. No problems until me and my friend are ready to leave camp and enter Act 3. We click leave camp, get the achievement for finishing Act 2 (I didn't have Achievement enabler on my previous runs), and I load to 100%, hear the background ambiance as if our characters are loaded into the area, and proceed to crash after 5-10 seconds. My friend, however, loads into Wyrm's Crossing for a few seconds and is able to look around for a little before the server shuts down (I am hosting). 

Steps Taken To Troubleshoot:
Kicking Party Characters to Camp to have 4 total members (Originally 6), and even just the 2 Avatars, (Friend is playing Durge, cannot kick him to camp).
Removing all equipped items and respec'ing to lvl 1 for all members.
Removed all possible mods that I can while still being able to load the save.
Updated every mod, Reinstalled every mod.
Reinstalled SE.
Tried older versions of SE.
Tried "Devel" releases for SE.
Cleared as many cache and temp files as I could find.
Sent the save file to my friend to have him host.
Reinstalled the game, then everything above ^ again.

Notes:
This problem persists even if I load from BEFORE the Ketheric fight in Act 2.
I have tested every save point between then and right before clicking the leave camp button.
The problem also exists for my friend if he is the one hosting the game.

Any help or suggestions on the matter are welcome, this is a save game with 92+ hours on it and I would hate to have to restart.
Thank you in advance for taking the time to help <3

### Indications

The only indicator that points me to script extender is in the CrashDump files produced when the game closes, is the first line after analyzing the crash is "*** WARNING: Unable to verify checksum for BG3ScriptExtender.dll". 
I'm not familiar enough with WinDbg to see if this is the actual cause of the problem.
Changing/Removing as many mods as possible has given no indication of being the issue, and typically has caused more problems.

### Diagnostic Files

Diagnostic Tool Data: 
[BG3 Diagnostic Data.txt](https://github.com/user-attachments/files/15745863/BG3.Diagnostic.Data.txt)

Most Recent Crash Dump:
[CrashDump - 06-07-2024 20h 38m 01s.dmp](https://github.com/user-attachments/files/15745865/CrashDump.-.06-07-2024.20h.38m.01s.dmp)

SE Runtime Log:
[Extender Runtime 2024-06-07 20-36-12.log](https://github.com/user-attachments/files/15745867/Extender.Runtime.2024-06-07.20-36-12.log)

Mod Load Order (BG3MM):
[Current Load Order.json](https://github.com/user-attachments/files/15745876/Current.Load.Order.json)

Osiris Log is too large to upload here, but in case it's needed I will happily upload it to either a google drive or dropbox and provide the link if needed in the comments. 

### Steps to Reproduce

1. Finish Act 2
2. Click "Road to Baldur's Gate"
3. Complete Dialogue as Intended (Night)
4. Long Rest to Begin Astral Plane Event
5. Finish Astral Plane Event/Finish Long Rest
6. Complete Dialogue as Intended (Morning)
7. Attempt to leave camp
8. Crash

### Expected Behavior

Upon leaving camp, Act 3 should begin and the party should be taken to Wyrm's Crossing. 

### Actual Behavior

Game will load to 100%, with audio in the background as if the party is loaded in (birds chirping and distant talking), then proceed to close/crash after 5-10 seconds.

**Comment by Norbyte** [comment]
Hi,

I've checked the crash dump and this seems to be a crash somewhere inside the entity world update logic of the game, not SE.
I don't have debug symbols for the .exe so I'm unable to tell what exactly is causing the issue.

**Comment by FrenchiestOfFries** [comment]
Thanks for your reply! Turns out the issue revolved around the Dual Wield Overhaul mod I had installed, I assumed it wasn't going to be the problem as I wasn't able to launch the save without it, but when I reset every party member to lvl 1 and unequipped everything, it allowed me to launch the save, get into Wyrm's Crossing, save again, and reinstall the mod. Apologies for the unnecessary issue post, but at the very least it was helpful to know it wasn't SE related and gave me some willpower to go back to trial and error troubleshooting. Thanks again for taking the time to look through my issue and respond though!
